modules/migration-mtc-workflow.adoc

```diff
@@ -4,12 +4,10 @@
 // * migration/migrating_4_1_4/migrating-application-workloads-4-1-4.adoc
 // * migration/migrating_4_2_4/migrating-application-workloads-4-2-4.adoc
 
-[id='migration-understanding-cam_{context}']
-= About the {mtc-full}
+[id='migration-mtc-workflow_{context}']
```

Comment:
If you're renaming a file and backporting to 4.5 and 4.6, this will require redirects. Have those been requested?

Reply:
I don't understand. I thought I was changing the header, not the file name. Am I missing something?

Reply:
@apinnick if you look at the the top of this file in the diff, it shows that you've renamed the file from `migration-understanding-cam.adoc` to `migration-mtc-workflow.adoc`. See the screenshot below. Did you not mean to do that?

![image](https://user-images.githubusercontent.com/7443887/106652324-49d81e00-6563-11eb-9afe-7c09f57ff087.png)

Also, even if you're just changing the title of the module, you do want to try avoid changing the anchor (if possible) on it if it's being backported to a published version, because then anyone with those old links saved won't work.

Reply:
@bergerhoffer  
Now I see. Yes, I changed a module file name. Vikram said it was OK to change a module file name but not an assembly file name.

Changing a module name does not cause the link to break. If the anchor link changes (the part after `#`, the link resolves to the assembly URL, which is the HTML page. There is no 404. The information that the user was looking for (diagram of workflow) is still there, but with a clearer heading. This change should not cause any problems for the user.

Reply:
@vikram-redhat are we only setting up redirects upon assembly name changes that would cause a 404, and not module name changes?

Reply:
@bergerhoffer correct. Module name changes don't require redirects. I will update the manual to clarify this.

Reply:
Cool thanks for clarifying!